src/makeRobot.js

```diff
@@ -39,6 +39,86 @@
  */
 function makeRobot(name, wheels, version) {
   // write code here
+  const robot = {
+    name,
+    wheels,
+    version,
+    coords: {
+      x: 0,
+      y: 0,
+    },
+    get info() {
+      return 'name: '
+      + this.name
+      + ', chip version: '
+      + this.version
+      + ', wheels: '
+      + this.wheels;
+    },
+    get location() {
+      return `${this.name}: x=${this.coords.x}, y=${this.coords.y}`;
+    },
+    goForward(times = 0) {
```

Comment:
```suggestion
    goForward(step = 1) {
```
I think step sounds better.
Also, why do you need to make 0 a default value?
Make it 1 and in that case you will need to make only one check